genie-web/src/main/java/com/netflix/genie/web/properties/LocalAgentLauncherProperties.java

```diff
@@ -45,11 +45,42 @@
      */
     public static final String PROPERTY_PREFIX = "genie.agent.launcher.local";
 
+    /**
+     * Placeholder for server port in command-line-template.
+     */
+    public static final String SERVER_PORT_PLACEHOLDER = "<SERVER_PORT_PLACEHOLDER>";
```

Comment:
These strings do not match the documentation you put in. You don't have the `<>` there